src/Console/InstallCommand.php

```diff
@@ -31,6 +31,6 @@ public function handle()
     {
         $this->call('passport:keys', ['--force' => $this->option('force'), '--length' => $this->option('length')]);
         $this->call('passport:client', ['--personal' => true, '--name' => config('app.name').' Personal Access Client']);
-        $this->call('passport:client', ['--password' => true, '--name' => config('app.name').' Password Grant Client']);
+        $this->call('passport:client', ['--password' => true, '--name' => config('app.name').' Password Grant Client', '--provider' => array_keys(config('auth.providers'))[0]]);
```

Comment:
Is it sensible to pick the first one here?

Reply:
Maybe not, I wasn't really sure what the best solution was. We could hardcode the default to "users" since that ships with Laravel. The right approach might be to attempt to use "users" if its an actual provider, otherwise default to null.

Thoughts?

Reply:
I can follow that.